path_planning/launch/slam.launch

```diff
@@ -2,6 +2,8 @@
 
   <arg name="lidar_type" default="4"/>
   <arg name="imu_topic" default="/livox/imu"/>
+  <arg name="min_alt" default="1.5"/>
```

Comment:
This is quite different from 0.2 and 1.3, are we sure we want this to be the range? I'm not sure I fully understand what the altitude limit is for either

Reply:
Yep, it's the limits the path searcher will allow for finding obstacle free paths. And it's in the units of the map, so this is to range from 1.5 to 4 m by default, but we will need to verify actual subcanopy range based on the site